e2e-tests/functions

```diff
@@ -1288,8 +1290,11 @@ function deploy_operator_gh() {
 	fi
 	apply_rbac_gh "${rbac_yaml}" "${git_tag}"
 	curl -s "https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/${git_tag}/deploy/${operator_yaml}" >"${tmp_dir}/${operator_yaml}_${git_tag}.yaml"
-	$sed -i -e "s^image: .*^image: ${IMAGE}^" "${tmp_dir}/${operator_yaml}_${git_tag}.yaml"
-	kubectl_bin apply -f "${tmp_dir}/${operator_yaml}_${git_tag}.yaml" ${OPERATOR_NS:+-n $OPERATOR_NS}
+
```

Comment:
@pooknull please check psmdb operator, we have added tests cases for this functionality you need just copy it to PXCO 
https://github.com/percona/percona-server-mongodb-operator/blob/main/e2e-tests/version-service/run

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1236/commits/fafaa07baa9c168fa38eb8af9360c89a7d828606